### ISSUE TITLE
ui.plotly(fig) gives TypeError: unsupported datatype in numpy array

### ISSUE THREAD

**Issue by JS-Aibel** [zauberzeug/nicegui#601]
Hi,

The example below results in an error message saying “TypeError: unsupported datatype in numpy array”. The plotly example is taken from https://plotly.com/python/radar-chart/ and works with fig.show(). Could this be related to https://github.com/zauberzeug/nicegui/discussions/517 ?

```py
df = pd.DataFrame(dict(
r=[1, 5, 2, 2, 3],
theta=['processing cost','mechanical properties','chemical stability', 'thermal stability', 'device integration']))

fig = px.line_polar(df, r='r', theta='theta', line_close=True)
fig.update_traces(fill='toself')
ui.plotly(fig)
```

**Comment by rbeeli** [comment]
@falkoschindler 
We can add a manual call to `arr.tolist()` for `np.array` types within our `json` serializers, this would make the interface for plotting much more user friendly because `np.array([], dtype=object)` occur quite frequently. Will have a look on the weekend.

Sidenote: I think we should add more examples of Plotly charts (incl. the two ways of how to specify them) in the docs page, currently there is only one. Probably would be beneficial for other components as well to have more examples than just one.

**Comment by rbeeli** [comment]
@falkoschindler, see PR: [https://github.com/zauberzeug/nicegui/pull/669](https://github.com/zauberzeug/nicegui/pull/669). Also added additional example for Plotly in docs page.

**Comment by falkoschindler** [comment]
Great! I just merged @rbeeli's PR #669. As far as I can tell, it fixes this issue.
Thank you, @rbeeli and @JS-Aibel for your continued contributions!
